lms/djangoapps/student_account/views.py

```diff
@@ -170,72 +163,43 @@ def _third_party_auth_context(request):
     """
     context = {
         "currentProvider": None,
-        "providers": []
+        "providers": [],
+        "finishAuthUrl": None,
+        "errorMessage": None,
     }
 
-    course_id = request.GET.get("course_id")
-    email_opt_in = request.GET.get('email_opt_in')
-    redirect_to = request.GET.get("next")
-
-    # Check if the user is trying to enroll in a course
-    # that they don't have access to based on country
-    # access rules.
```

Comment:
@rlucioni Yes, could you please open a bug for that internally? I'm on a tight schedule for the SSO work and I agree that since it's an existing issue it's outside the scope of this PR. Thanks!